### ISSUE TITLE
Crashes when opening "today" view.

### ISSUE THREAD

**Issue by tzugen** [alainm23/planify#911]
I am syncing with an existing todo list via CalDav.
When I click on "Today" the app crashes.

Log report:

```
ERROR:../src/Services/CalDAV/Util.vala:134:cal_dav_util_ical_to_date_time_local: assertion failed: (!date.is_null_time ())
Bail out! ERROR:../src/Services/CalDAV/Util.vala:134:cal_dav_util_ical_to_date_time_local: assertion failed: (!date.is_null_time ())

```

**Comment by catdevnull** [comment]
Same thing happens to me. Using Radicale with DAVx5 in my phone with [tasks.org](https://tasks.org)

**Comment by 0chroma** [comment]
same issue and error, I'm finding this happens when going to "scheduled" as well. Happy to export a task for debugging if it'd help!

**Comment by 0chroma** [comment]
Here's a sample task I made in tasks.org in order to help debugging: [test task.txt](https://github.com/alainm23/planner/files/10932484/test.task.txt)

it's an ics file but github won't allow me to upload those so you can just change the extension after downloading

currently this is the only thing stopping me from using planner so let me know if I can help any other way

**Comment by alainm23** [comment]
Fixed in Planify 4.1
